Database, Login and SessionManagement

paaddyy opened the pull request.

Commit:
Database, Login and SessionManagement

wasamasa commented on src/de/bg/fhdw/bfwi413a/karthago/activities/login/ApplicationLogic.java:
```diff
@@ -1,21 +1,30 @@
 
 import de.bg.fhdw.bfwi413a.karthago.Navigation;
-import android.util.Log;
+import de.bg.fhdw.bfwi413a.karthago.session_management.SessionManagement;
```
Das sieht einfach nur verkehrt aus. Snake Case _und_ Camel Case?

paaddyy replied:
Ok, ich änder es gleich noch ;-)

wasamasa replied:
Oh und auch wenn es ein Paket Session Management geben muss, eine Klasse mit dem gleichen Namen hinzuzufügen sieht aus als ob entweder das Paket nicht notwendig ist oder der Klassenname nicht ideal ist.

paaddyy replied:
Hab ich jetzt auch verändert :)

Commit:
Completed SessionManagement

SessionManagement fully integrated and unnecessary things deleated

Commit:
Working UserChanging and UserDeleating Process

UserChanging and UserDeleating Process finally works. All cases (never
deleate User "ADMIN", deleating actual User) are working fine.

wasamasa commented:
OK, das sieht jetzt viel besser aus.
